src/scripts/main.js

```diff
@@ -1,7 +1,24 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const notification = document.createElement('div');
+  const notificationTitle = document.createElement('h2');
+  const notificationDescription = document.createElement('p');
+  const page = document.querySelector('body');
+
+  notification.className = `notification ${type}`;
+  notificationTitle.textContent = title;
+  notificationTitle.className = 'title';
+  notificationDescription.textContent = description;
+
+  page.append(notification);
+  notification.append(notificationTitle);
+  notification.append(notificationDescription);
+
+  notification.style.top = `${posTop}px`;
+  notification.style.right = `${posRight}px`;
```

Comment:
Better append message when we add all content and styles

```suggestion
  notification.append(notificationTitle);
  notification.append(notificationDescription);

  notification.style.top = `${posTop}px`;
  notification.style.right = `${posRight}px`;
  
  page.append(notification);
```